assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.js

```diff
@@ -105,7 +111,13 @@ function AudienceSegmentationSetupCTAWidget( { Widget, WidgetNull } ) {
 
 	const { apiErrors, failedAudiences, isSaving, onEnableGroups } =
 		useEnableAudienceGroup( {
-			onSuccess: () => {
+			onSuccess: async () => {
+				await dismissPrompt(
```

Comment:
Hmm, maybe we should _not_ make this async so the dismissal requests can potentially fire in parallel:

```suggestion
			onSuccess: () => {
				dismissPrompt(
```